### ISSUE TITLE
TASK: NumPy 2 is missing from ELN

### ISSUE THREAD

**Issue by hroncok** [fedora-eln/eln#215]
### What does the ELN SIG need to do?
* https://fedoraproject.org/wiki/Changes/NumPy2
* numpy 2 has landed in rawhide
* the ELN rebuild failed (I tried a manual one and it failed as well)
* the rawhide build was not tagged to the ELN buildroot
* [some rawhide packages need a rebuild (e.g. scipy)](https://bugzilla.redhat.com/show_bug.cgi?id=2332408)

Now if we go bump and rebuild the packages for NumPy 2, they will be rebuilt with NumPy 1 in ELN. Once NumPy 2 finally reahces ELN, new ELN rebuilds will be necessary.

I have no idea if this is an isolated problem (scipy only) or if there will be more such cases.

**Comment by yselkowitz** [comment]
See https://bugzilla.redhat.com/show_bug.cgi?id=2332211

**Comment by hroncok** [comment]
Does that mean we cannot even tag the rawhide's numpy in because it's not binary compatible?

Should I hold off scipy rebuild or do it?

**Comment by yselkowitz** [comment]
No, the focus should be on fixing numpy.  Once done, then its dependents can be rebuilt; a dist-tag bump will probably be necessary to avoid a bunch of ELN-specific revbumps.

**Comment by yselkowitz** [comment]
@hroncok numpy 2.2.0 has now been built for ELN.  Could you advise as to which packages now need to be rebuilt again as a result?

**Comment by hroncok** [comment]
I only know about scipy.

**Comment by yselkowitz** [comment]
scipy has been rebuilt, as have all of its reverse dependencies

**Comment by hroncok** [comment]
Thanks.
